libbeat/scripts/Makefile

```diff
@@ -57,13 +68,13 @@ endif
 ### BUILDING ###
 
 # Builds beat
-.PHONY: build
-build: $(GOFILES)
+.PHONY: ${BEATNAME}
+${BEATNAME}: $(GOFILES)
```

Comment:
Yes, this change kind of does not belong in this PR. As build is normally quite fast, I think it is not too big of an issue that it is always built, especially as having the right implementation with all the files changed is tricky. I suggest we remove $(GOFILES) in a follow up PR.